Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -249,6 +255,15 @@ open class MapView: UIView {
             mapboxMap.setCamera(to: cameraOptions)
         }
 
+        insertSubview(viewAnnotationContainerView, at: 1)
```

Comment:
Would be less brittle to insert the subview relative to some other view (e.g. above the metalView)